src/simulator/simulator.cpp

```diff
@@ -25,13 +25,28 @@ namespace mrover {
 
         mIkTargetPub = mNh.advertise<IK>("/arm_ik", 1);
 
-        initWindow();
+        mIsHeadless = mPnh.param<bool>("headless", false);
+        mEnablePhysics = mIsHeadless;
+        {
+            XmlRpc::XmlRpcValue gpsLinearization;
+            mNh.getParam("gps_linearization", gpsLinearization);
+            if (gpsLinearization.getType() != XmlRpc::XmlRpcValue::TypeStruct) throw std::invalid_argument{"GPS lineraization must be a struct. Did you rosparam load a localization config file properly?"};
```

Comment:
typos on the word linearization. also why are you using xmlrpc instead of just getting them as doubles? to avoid putting gps_linearization in front of all of them?

Reply:
Tbh I didn't know I could do it the other way

Reply:
yeah you can do it directly with common primitives and common containers of primitives, but the ros docs are out of date so if you google it you have to scroll down to find it

Reply:
Cool. I think the original reason was to avoid prefixing all of them

Reply:
still typo

Reply:
ditto

Reply:
Fine